app/src/main/java/fr/free/nrw/commons/upload/ImageProcessingService.java

```diff
@@ -56,77 +68,94 @@ public ImageProcessingService(FileUtilsWrapper fileUtilsWrapper,
         Single<Integer> wrongGeoLocation = checkImageGeoLocation(uploadItem.getPlace(), filePath);
         Single<Integer> darkImage = checkDarkImage(filePath);
         Single<Integer> itemTitle = checkTitle ? validateItemTitle(uploadItem) : Single.just(ImageUtils.IMAGE_OK);
+        Single<Integer> checkFBMD = checkFBMD(filePath);
 
-        return Single.zip(duplicateImage, wrongGeoLocation, darkImage, itemTitle,
+        Single<Integer> zipResult = Single.zip(duplicateImage, wrongGeoLocation, darkImage, itemTitle,
                 (duplicate, wrongGeo, dark, title) -> {
                     Timber.d("Result for duplicate: %d, geo: %d, dark: %d, title: %d", duplicate, wrongGeo, dark, title);
                     return duplicate | wrongGeo | dark | title;
                 });
+
+        return Single.zip(zipResult, checkFBMD, (zip, fbmd) -> {
+            Timber.d("zip:"+zip+ "fbmd:"+zip);
+            return zip | fbmd;
+        });
     }
 
-    /**
-     * Checks item title
-     * - empty title
-     * - existing title
-     * @param uploadItem
-     * @return
-     */
-    private Single<Integer> validateItemTitle(UploadModel.UploadItem uploadItem) {
-        Timber.d("Checking for image title %s", uploadItem.getTitle());
-        Title title = uploadItem.getTitle();
-        if (title.isEmpty()) {
-            return Single.just(EMPTY_TITLE);
+        public Single<Integer> checkFBMD(String filePath){
```

Comment:
Please add method Javadoc explaining the whole reasoning behind this issue (what FBMD is, and how images from Facebook are often copyright violations).

Reply:
Also added a documentation here :+1: